qemu/tests/cfg/qemu_img.cfg

```diff
@@ -90,9 +90,39 @@
         - rebase:
             no raw vmdk qed
             subcommand = rebase
-            rebase_mode = unsafe
-            image_name_snapshot1 = sn1
-            image_name_snapshot2 = sn2
+            variants:
+                - unsafe:
+                    rebase_mode = unsafe
+                - show_progress:
+                    command_result_pattern = "\(\d+\.\d+/100%\)"
+                    variants:
+                        - show_progress_on:
+                            show_progress = on
+                            check_output = stdout
+                        - show_progress_off:
+                            show_progress = off
+                            check_output = stderr
+                            cache_mode = directsync
+                - new_to_base:
+                    backing_file_snapshot2 = ""
+                    check_output = stderr
+                    command_result_pattern = "Could not open old backing file ''"
+                    skip_boot = yes
+                    variants:
+                        - on_iscsi:
```

Comment:
there will be more scenerios under new_to_base?
if no, you can update to new_to_base_iscsi: